app/src/main/java/com/duckduckgo/app/browser/BrowserActivity.kt

```diff
@@ -437,7 +437,9 @@ open class BrowserActivity : DuckDuckGoActivity() {
         currentTab?.tabId?.let {
             val params = PrivacyDashboardHybridWithTabIdParam(it)
             val intent = globalActivityStarter.startIntent(this, params)
-            startActivityForResult(intent, DASHBOARD_REQUEST_CODE)
+            intent?.let {
```

Comment:
Is this part of the intended change?

Reply:
Incidentally, `startIntent()` can return null, so this could crash.
I can remove it though and let it crash (?)